cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -85,13 +85,14 @@ spec:
   # enable the crash collector for ceph daemon crash collection
   crashCollector:
     disable: false
-  cleanupPolicy:
-    # cleanupPolicy should only be added to the cluster when the cluster is about to be deleted.
+  cleanup:
```

Comment:
Need to revert this, right? I think you intended to change the instance on line 143.

Reply:
No I thought you wanted to remove the Policy word...

Reply:
oh actually yes! my bad